File: cpp/23c_723A_TheNewYearMeetingFriends.cpp
#include<bits/stdc++.h>

int main(){
    std::vector<int> dist;
    int temp;
    for(int i = 0; i < 3; i++){
        std::cin>>temp;
        dist.push_back(temp);
    }
    sort(dist.begin(),dist.end());
    int ans = (dist[1] - dist[0]) + (dist[2] - dist[1]);
    std::cout<<ans;
    return 0;
}

File: python/23c_723A_TheNewYearMeetingFriends.py
def main():
    dist = input().split()
    for i in range(len(dist)):
        dist[i] = int(dist[i])
    dist.sort()
    ans = (dist[1] - dist[0]) + (dist[2] - dist[1])
    print(ans)

if __name__ == "__main__":
    main()